hexrdgui/hexrd_config.py

```diff
@@ -1002,6 +1009,13 @@ def intensity_corrected_images_dict(self):
             for name, img in images_dict.items():
                 images_dict[name] = img - minimum
 
+        if HexrdConfig().apply_median_filter_correction:
+            for name, img in images_dict.items():
+                images_dict[name] = medfilt2d_memoized(
+                    img,
+                    kernel_size=HexrdConfig().median_filter_kernel_size
+                )
```

Comment:
I'll need to remember to move this with the other intensity corrections in my other PR...